src/event/trigger.js

```diff
@@ -93,7 +96,7 @@ jQuery.extend( jQuery.event, {
 		// Fire handlers on the event path
 		i = 0;
 		while ( ( cur = eventPath[ i++ ] ) && !event.isPropagationStopped() ) {
-
+			lastElement = cur;
```

Comment:
@gibson042 in my opinion, it would reduce the size at a high cost of code readability. What is the code guideline in the `jQuery` project with regard to readability v.s. size?